docs/ref/models/class.txt

```diff
@@ -11,6 +11,35 @@ reference guides </ref/models/index>`.
 Attributes
 ==========
 
+``DoesNotExist``
+----------------
+
+.. exception:: Model.DoesNotExist
+
+    This exception is raised by the ORM in a couple places, for example by
+    :meth:`QuerySet.get() <django.db.models.query.QuerySet.get>` when an object
+    is not found for the given lookups.
```

Comment:
You mention that this is raised in a couple of places (couple = two) and then only mention one. I think you could drop the "couple" bit and mention more/all places.

I found ``QuerySet.get()``, ``Model._get_next_or_previous_by_FIELD()`` (which only seems to be used by ``DateField`` and subclasses) and I think in the handling of relationships for `OneToOneField`. (It also seems that there is a - non-documented - subclass `RelatedObjectDoesNotExist`.)

Reply:
I copied this over and I believe it's the informal "couple" meaning "several". Yes will update to list the other places, and check for links from their documentation to here.

Reply:
`RelatedManager.RelatedObjectDoesNotExist` is a subclass of `Model.DoesNotExist` - added docs under `OneToOneField`, which were incorrect since they showed a plain `DoesNotExist` being raised.